src/components/property/PropertyList.js

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
+import { connect, useStore } from 'react-redux';
 import PropertyTile from './PropertyTile';
 
-const PropertyTileBox = ({ data }) => {
+const PropertyList = ({ properties }) => {
+  const store = useStore();
+  // console.log(properties);
 
-  if (!data) return null;
-
-  const properties = data.map((property, index) => {
+  const listResultProperties = properties.map((property, index) => {
     return (
-      <PropertyTile key={index} data={property} />
+        <PropertyTile key={ index } { ...property } />
     )
+    
   })
 
   return (
-    <div className="property-list">
-      {properties}
-    </div>
+      <div className="property-list">
+          {listResultProperties}
+      </div>
   )
 }
 
-export default PropertyTileBox;
+const mapStateToProps = state => ({ 
+  properties: state.properties.properties.results
+  // TODO: add errors here
+})
```

Comment:
There's something weird happening where state from the store here (i.e. the state being passed as a param) is not persisting. I keep having to go back to the landing and choose another city in order to progress.